alicloud/provider.go

```diff
@@ -175,6 +175,7 @@ func Provider() terraform.ResourceProvider {
 			"alicloud_fc_services":                       dataSourceAlicloudFcServices(),
 			"alicloud_fc_triggers":                       dataSourceAlicloudFcTriggers(),
 			"alicloud_db_instances":                      dataSourceAlicloudDBInstances(),
+			"alicloud_db_instances_sql_log":              dataSourceAlicloudDBInstancesSQLLog(),
```

Comment:
please use `alicloud_db_instance_sql_log_records` instead of  `alicloud_db_instances_sql_log `